### PR TITLE
BPK-870 react native docs

### PR DESCRIPTION
**Pull request by matthewdavidson**
+ [x] For any new components I've made sure that the style can be extended by the consumer using a `className` override.
+ [x] For any new files I've included the [Apache 2.0 License header](https://github.com/Skyscanner/backpack/blob/master/packages/bpk-tokens/formatters/license-header.js#L2-L16) at the top of the file.
+ [ ] I've updated `changelog.md` for any changes that need to be highlighted in the changelog.
+ [x] If I've updated `npm-shrinkwrap.json` those changes are intentional.

### CODE REVIEW

File: native/react-native-bpk-component-text/index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import BpkText from './src/BpkText';
```
Comment (georgegillams) [inline native/react-native-bpk-component-text/index.js]: Shouldn't all new files have the license header?

File: packages/bpk-docs/src/components/Header/Header.js [hunk 2]
```diff
@@ -35,7 +35,8 @@ const getClassName = cssModules(STYLES);
 const headerLinks = [
   { to: ROUTES.USING_BACKPACK, children: 'Using Backpack' },
   { to: ROUTES.STYLE, children: 'Style' },
-  { to: ROUTES.DOCS, children: 'Components' },
+  { to: ROUTES.COMPONENTS, children: 'Components' },
+  // { to: ROUTES.NATIVE, children: 'Native' },
```
Comment (georgegillams) [inline packages/bpk-docs/src/components/Header/Header.js]: Is this needed or not?

File: native/react-version-test.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+it('native/* should be ^16.0.0', () => {
```
Comment (k0nserv) [inline native/react-version-test.js]: Should we keep these?

File: packages/react-version-test.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+it('packages/* should be ^15.0.0', () => {
+  expect(React.version).toMatch(/^15/);
```
Comment (k0nserv) [inline packages/react-version-test.js]: Ditto

File: native/react-version-test.js (hunk 3 above)
Comment (matthewdavidson) [reply native/react-version-test.js]: So I pondered this too ... whilst the shrinkwrap determines what version we use, it does not guarantee that node resolves the right version. I kinda like them. They are cheap and will act as a failsafe.

File: packages/bpk-docs/src/components/Header/Header.js (hunk 2 above)
Comment (matthewdavidson) [reply packages/bpk-docs/src/components/Header/Header.js]: Yeah this is the "hide" part.

File: native/react-native-bpk-component-text/index.js (hunk 1 above)
Comment (matthewdavidson) [reply native/react-native-bpk-component-text/index.js]: Good catch, fixed

File: native/react-version-test.js (hunk 3 above)
Comment (k0nserv) [reply native/react-version-test.js]: That shounds good to me :)

**Comment by matthewdavidson** [comment]
@georgegillams you happy?

**Pull request merged**